extensions/wikia/UserProfilePageV3/models/FavoriteWikisModel.class.php

```diff
@@ -27,11 +27,13 @@ class FavoriteWikisModel extends WikiaModel {
 	 * @param User $user
 	 */
 	public function __construct( User $user ) {
+		parent::__construct();
```

Comment:
Minor: Is there a reason to call the parent constructor here? It doesn't look like we're using any of the helper values it initializes in this PR or in the other one.